src/Sylius/Bundle/InventoryBundle/EventListener/StockItemListener.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Sylius sandbox application.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\InventoryBundle\EventListener;
+
+use Sylius\Component\Inventory\Factory\StockItemFactoryInterface;
+use Symfony\Component\EventDispatcher\GenericEvent;
+
+/**
+ * Creates stock items for new location or stockable.
+ *
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class StockItemListener implements StockItemListenerInterface
+{
+    /**
+     * @var StockItemFactoryInterface
+     */
+    private $stockItemFactory;
+
+    /**
+     * @param StockItemFactoryInterface
+     */
+    public function __construct(StockItemFactoryInterface $stockItemFactory)
+    {
+        $this->stockItemFactory = $stockItemFactory;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function onStockableCreate(GenericEvent $event)
```

Comment:
We usually name listener methods by what they do rather then by event names (e.g.: `OrderNumberListener::generateOrderNumber()`, `OrderUpdateListener::recalculateOrderTotal()`...).